lib/xcode.js

```diff
@@ -269,6 +269,44 @@ const simulatorDevicePairCompatibility = {
 			'9.x': true,		// watchOS 9.x
 			'10.x': true		// watchOS 10.x
 		}
+	},
+	'16.x': {                   // Xcode 16.x
+		'13.x': {               // iOS 13.x
+			'8.x': true,        // watchOS 8.x
+			'9.x': true,		// watchOS 9.x
+			'10.x': true,		// watchOS 10.x
+			'11.x': true,		// watchOS 11.x
+		},
+		'14.x': {               // iOS 14.x
+			'8.x': true,        // watchOS 8.x
+			'9.x': true,		// watchOS 9.x
+			'10.x': true,		// watchOS 10.x
+			'11.x': true,		// watchOS 11.x
+		},
```

Comment:
```suggestion
```

I dunno about iOS 13 and 14... the oldest iOS version Xcode 16 beta can install is iOS 15. You sure about iOS 13 and 14?

Reply:
My concerns is 2 fold. Do we even support iOS 13 and 14? Given we can't install iOS 13 or 14, I would have to assume they aren't supported by Xcode 16 (or 15). Newer versions of ioslib have a bin that can detect device pairs, but it only uses the installed SDKs, so it's really no help here.

My vote is to drop 13 and 14 simulator support. I'll approve to unblock this, so it's up to you if you wanna nuke them now or later.

Reply:
This is just for the Simulator mappings. Titanium itself supports iOS 14+, but we kept older versions, as you argued in earlier releases that ioslib could still be used aside of new titanium-sdk releases where it's included. I would vote for removing iOS 13 to align with the minimum iOS version in titanium-sdk and go from there.

Reply:
we should also update https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Titanium_Compatibility_Matrix/#supported-versions-of-ios-sdk-target-ios-platform to reflect that. Currently it still says min iOS 12 and Xcode 12

+1 for removing iOS 13 if it is not supported